src/lib.rs

```diff
@@ -2377,6 +2444,108 @@ mod tests {
         }
     }
 
+    #[test]
+    fn tensor_debug() {
+        let values: Vec<i32> = (1..9).collect();
+        let tests = [
+            (
+                "Tensor<i32> { values: [1, 2, 3, 4, 5, 6, 7, 8], dtype: int32, shape: [8]}",
```

Comment:
This format has spaces around the opening curly brace, but not the last one, which is inconsistent.